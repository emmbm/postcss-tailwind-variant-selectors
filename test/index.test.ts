import { describe, it, vi } from 'vitest';

describe('plugin build output', async () => {
  it('parses @variant rules and transforms :variant() selectors', async (ctx) => {
    const { default: css } = await vi.importActual('index.css');
    ctx.expect(css).toBeDefined();
    ctx.expect(String(css)).toMatchSnapshot();
  });
});
