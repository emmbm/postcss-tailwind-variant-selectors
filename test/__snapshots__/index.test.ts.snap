// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`plugin build output > parses @variant rules and transforms :variant() selectors 1`] = `

@layer components {
  .valid {
    color: black;
    &:hover {
      color: red;
    }
    &::after {
      color: pink;
    }
  }
}

`;
